lib/ui/myplot/my_plot_page.dart

```diff
@@ -2,10 +2,99 @@ import 'package:farmsmart_flutter/data/model/crop_entity.dart';
 import 'package:farmsmart_flutter/redux/app/app_state.dart';
 import 'package:farmsmart_flutter/redux/home/myPlot/my_plot_actions.dart';
-import 'package:farmsmart_flutter/ui/myplot/my_plot_child_item.dart';
 import 'package:farmsmart_flutter/ui/myplot/myplot_viewmodel.dart';
+import 'package:farmsmart_flutter/utils/strings.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_redux/flutter_redux.dart';
+import 'new_my_plot_child_item.dart';
+
+abstract class HomeMyPlotPageStyle {
+
+  final Color iconButtonColor;
+  final Color primaryGreen;
+  final Color separatorWhite;
+
+  final EdgeInsets edgeInsetsTop;
+  final EdgeInsets edgeInsetsTitle;
+  final EdgeInsets listPadding;
+  final EdgeInsets paddingBetweenElements;
+  final EdgeInsets paddingBeforeCropDayCount;
+  final EdgeInsets paddingForDayCount;
+  final EdgeInsets separatorIndent;
+
+  final BorderRadius ovalRadiousForDayCount;
+
+  final Alignment circularProgressIndicatorAligmentCenter;
+
+  final MainAxisAlignment mainAxisAlignmentSpaceBeetwen;
+  final MainAxisAlignment mainAxisAlignmentSpaceStart;
+  final MainAxisSize mainAxisSizeMin;
+
+  final CrossAxisAlignment crossAxisAlignmentStart;
+
+
+  final TextStyle titleTextStyle;
+  final TextStyle cropNameTextStyle;
+  final TextStyle cropStatusTextStyle;
+  final TextStyle cropDayTextStyle;
+
+  final double buttonShapeRadious;
+  final double buttonIconSize;
+  final double oppacityForDayCount;
+  final double separatorHeight;
+  final double sizeForDayCountShape;
+
+  HomeMyPlotPageStyle(this.iconButtonColor, this.primaryGreen,
+      this.separatorWhite,
+      this.edgeInsetsTop, this.edgeInsetsTitle, this.listPadding,
+      this.paddingBetweenElements,this.paddingBeforeCropDayCount,
+      this.paddingForDayCount,
+      this.ovalRadiousForDayCount,
+      this.circularProgressIndicatorAligmentCenter,
+      this.mainAxisAlignmentSpaceBeetwen, this.mainAxisAlignmentSpaceStart,
+      this.mainAxisSizeMin,
+      this.crossAxisAlignmentStart, this.titleTextStyle, this.cropNameTextStyle,
+      this.cropStatusTextStyle, this.cropDayTextStyle, this.buttonShapeRadious,
+      this.buttonIconSize, this.oppacityForDayCount, this.separatorHeight, this.separatorIndent,
+      this.sizeForDayCountShape);
+}
+
+class _DefaultStyle implements HomeMyPlotPageStyle {
+
+  final Color iconButtonColor =  const Color(0xFFFFFFFF);
+  final Color primaryGreen =  const Color(0xff25df0c);
+  final Color separatorWhite = const Color(0xfff5f8fa);
+
+  final EdgeInsets edgeInsetsTop = const EdgeInsets.only(top: 20.0) ;
+  final EdgeInsets edgeInsetsTitle = const EdgeInsets.only(left: 25, top: 30, right: 5, bottom: 20);
+  final EdgeInsets listPadding = const EdgeInsets.only(left: 25.0, top: 25.0, right: 30.0, bottom: 25.0);
+  final EdgeInsets paddingBetweenElements = const EdgeInsets.only(bottom: 30);
+
+  final EdgeInsets paddingBeforeCropDayCount = const EdgeInsets.only(bottom: 23);
```

Comment:
I think some of these styles are for to list items, which are another widget. I think they should be moved to that widget and this style will just need an entry to provide a list item style. This will split this down and make it smaller.
The plot list Style should only contain the elements for its widget settings (like edge padding of the whole list, and the plus button style and title style)